### PR TITLE
UI styling of header component

### PR DESCRIPTION
**Pull request by parulraheja98**
I just was not sure about where Sign out button will go , I included it on far right of the screen. Rest mocks uploaded were very descriptive 

Refs: #214

### CODE REVIEW

File: web-ng/src/app/components/inline-edit-title/inline-edit-title.component.html [hunk 1]
```diff
@@ -21,5 +21,5 @@
   (blur)="updateProjectTitle($event)"
   AutoSizeInput
   class="inline-edit"
-  [includePlaceholder]="inclPlaceholder"
+  [minWidth]="424"
```
Comment (gino-m) [inline web-ng/src/app/components/inline-edit-title/inline-edit-title.component.html]: If this is static we shouldn't need data binding? i.e. `minWidth="424"`

File: web-ng/src/app/components/project-header/project-header.component.css [hunk 2]
```diff
@@ -14,12 +14,60 @@
...
+.logo {
+  margin-top: 16px;
+  margin-bottom: 16px;
+  margin-left: 20px;
+  vertical-align: middle;
+  width: 48px;
+  height: 48px;
+}
+
+.ground {
```
Comment (gino-m) [inline web-ng/src/app/components/project-header/project-header.component.css]: Please use more descriptive class name, e.g. `app-title`.

Also consider specifying the font as something like "Open Sans, Arial, sans-serif"

File: web-ng/src/app/components/project-header/project-header.component.html [hunk 3]
```diff
@@ -13,16 +13,23 @@
...
+    <span> Share </span>
+  </button>
+  <mat-icon class="icon-more-vert"> more_vert </mat-icon>
   <img
     src="{{ user.photoURL }}=s64"
     alt="{{ user.displayName }}"
     class="user-thumb"
   />
-  <button class="button" (click)="auth.signOut()">Sign out</button>
+  <button class="button btn-auth" (click)="auth.signOut()">Sign Out</button>
```
Comment (gino-m) [inline web-ng/src/app/components/project-header/project-header.component.html]: Fyi: These are temporary - they'll be replaced by #217 and #218.

File: web-ng/src/app/components/project-header/project-header.component.html [hunk 4]
```diff
@@ -13,16 +13,23 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-
+<img class="logo" src="/assets/img/logo.png" />
+<span class="ground"> Ground </span>
+<span> | </span>
```
Comment (gino-m) [inline web-ng/src/app/components/project-header/project-header.component.html]: Can we style this with CSS instead to make it look more polished and closer to the mocks?

File: web-ng/src/app/components/project-header/project-header.component.html [hunk 5]
```diff
@@ -13,16 +13,23 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-
+<img class="logo" src="/assets/img/logo.png" />
+<span class="ground"> Ground </span>
```
Comment (gino-m) [inline web-ng/src/app/components/project-header/project-header.component.html]: Please remove leading and trailing space in favor of CSS padding/margin.

File: web-ng/src/app/components/project-header/project-header.component.html [hunk 6]
```diff
@@ -13,16 +13,23 @@
...
+<span> | </span>
 <app-inline-edit-title></app-inline-edit-title>
 <div *ngIf="auth.user$ | async as user; else login" class="rhs-widgets">
+  <button class="btn-share">
+    <mat-icon class="icon-share"> share </mat-icon>
```
Comment (gino-m) [inline web-ng/src/app/components/project-header/project-header.component.html]: Is the text here the "alt text"? If so I assume the extra spaces are a stylistic choice?

File: web-ng/src/app/components/project-header/project-header.component.html [hunk 7]
```diff
@@ -13,16 +13,23 @@
...
+    <span> Share </span>
```
Comment (gino-m) [inline web-ng/src/app/components/project-header/project-header.component.html]: Same as above re CSS padding.

**Comment by parulraheja98** [comment]
@gino-m , Thanks for all the suggestions.
Attached are the screenshots:
image
image

File: web-ng/src/app/components/project-header/project-header.component.html (hunk 6 above)
Comment (parulraheja98) [reply web-ng/src/app/components/project-header/project-header.component.html]: Removed the extra spaces, share is just a type of icon it renders

**Pull request merged**